### DISCUSSION TITLE
keyboard is english, but app itself is 'mostly' in english but with some russian labels?

### DISCUSSION THREAD

**Discussion by CondomsPreventCoinkydinks** [AnySoftKeyboard/AnySoftKeyboard#3161] [Q&A] [answered]
think i've accidentally changed the app itself to use russian (not the keyboard language)? how do i change back to english?

https://i.imgur.com/XsAnYLm.jpg

thanks.

**Comment by menny** [comment, marked as answer]
This can happen either:
- your device locale is Russia
- you override the app locale (Anysoftkeyboard settings app home-> tweaks) to?Russian

**Reply by CondomsPreventCoinkydinks** [reply]
hi yeah, i forced locale to 'uk' thinking that was english, but it must be ukraine. sorted now. thanks bud.
